src/middlewares/partyMiddleware.js

```diff
@@ -0,0 +1,27 @@
+import validator from 'express-validator'
```

Comment:
I'm not sure you're supposed to use a `validator.` I think you're to write your validations yourself.